packages/plugins/rubygem-rainbow/rubygem-rainbow.spec

```diff
@@ -5,7 +5,7 @@
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 2.2.1
-Release: 4%{?dist}
+Release: 6%{?dist}
```

Comment:
Can we bump to 2.2.2 instead?

Reply:
I generally like it if there aren't irrelevant changes in the git log. Perhaps you can clean up the commits a bit.

Another piece of advice is that bumping is pretty safe while bump is its own discussion. If you submit the bumps as a separate PR it can be merged and not be held up by the revbump alignment discussion. Small PRs tend to be merged faster.